controllers/Frontend.php

```diff
@@ -10,52 +11,59 @@ public function home_cv()
     {
         $model_form = new Form();
         
-        $page="frontend/home_cv";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $formulaire = $model_form->form_page_home_cv();
+        $myView = new View('home_cv');
+        $myView->render();
     }
 
     public function legalnotice()
     {
-        $page="frontend/legalnotice";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('legalnotice');
+        $myView->render();
     }
     
     public function home()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
-
-        $page="frontend/home";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $manager = new ArticleManager();
+        $posts = $manager->get_posts_blog();
+        
+        $myView = new View('home');
+        $myView->render(array('posts' => $posts));
     }
 
     public function blog()
     {
         $model_article = new Article();
         $model_comment = new Comment();
+        $posts = $model_article->get_posts_blog_all();
         
-        $page="frontend/blog";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('blog');
+        $myView->render(array('posts' => $posts));
     }
 
     public function post()
     {
+        
         $model_article = new Article();
         $model_comment = new Comment();
                 
-        $page="frontend/post";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $responses = $model_comment->get_comments_blog();
+        $post = $model_article->get_article_blog();
+        $model_comment->form_comment_verification();
+        
+        if($post == false){
+            header("Location:/page-erreur");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)